pkg/apis/kubernikus/v1/secret.go

```diff
@@ -17,6 +17,8 @@ type Secret struct {
 	DexStaticPassword string `json:"dex-static-password,omitempty"`
 
 	Certificates
+
+	ExtraValues map[interface{}]interface{} `json:"extra-values,omitempty"`
```

Comment:
I'm not sure this will fly when reading this from the underlying Kubernetes secret.
The problem is secrets don't allow nested son structures. the field `extra-values` has to be of type `string` or `[]byte`. I think the serialisation, deserialisation will bomb for this struct.

I think what we need to do is make this a `string` field and `yaml.Unmarshal` it in `KlusterToHelmValues`.

Maybe we could also write a test that uses a kubernetes fake client to make sure this works.
You can probably add it the pkg/apis/kubernikus/v1/secret_test.go

Reply:
you are correct.

corrected.